src/nspawn/nspawn.c

```diff
@@ -3753,6 +3754,10 @@ static int setup_notify_child(const void *directory) {
         if (r < 0)
                 return log_error_errno(r, "SO_PASSCRED failed: %m");
 
+        r = setsockopt_int(fd, SOL_SOCKET, SO_PASSPIDFD, true);
+        if (r < 0)
+                log_debug_errno(r, "SO_PASSPIDFD failed, ignoring: %m");
```

Comment:
The log message sounds slightly borked.
Please say `"Failed to enable SO_PASSPIDFD, ignoring: %m"`, like the other places.
Please also update the above for SO_PASSCRED.